.github/workflows/release.yaml

```diff
@@ -117,6 +119,20 @@ jobs:
           pattern: release*
           path: releases
 
+      - name: Prepare universal binary
+        if: github.ref_type == 'tag' || (inputs.release_macos_amd64 && inputs.release_macos_arm64)
+        env:
+          MACOSX_UNIVERSAL_SUFFIX: "macosx-universal"
```

Comment:
Could we remove the `universal` suffix?
Unless it is a widely-accepted and idiomatic naming for MacOS apps.

Reply:
Removed.